releases/release-1.14/release-notes-draft.md

```diff
@@ -1,54 +1,113 @@
+## 1.14 What’s New
+
+
+Support for Windows Nodes is Going GA (#116)
```

Comment:
Unfortunately all of these have to be [#116](https://github.com/kubernetes/enhancements/issues/116) like:

```
[#116](https://github.com/kubernetes/enhancements/issues/116)
```

Or for k/k:

```
[#1234](https://github.com/kubernetes/kubernetes/issues/1234)
```

Reply:
I noticed that and will correct it. Thanks @marpaia